build.gradle

```diff
@@ -513,6 +520,17 @@ subprojects {
   }
 }
 
+gradle.projectsEvaluated {
+  subprojects {
+    plugins.withType(JavaPlugin) {
+      getAllDependentProjects(project).each {
+        tasks.javadoc.options.linksOffline "http://linkedin.github.io/gobblin/javadoc/${javadocVersion}/${it.name}/",
```

Comment:
slightly orthogonal to this PR, but any idea why we would need an offline line link to a previously published version of the Gobblin JavaDocs?